### PR TITLE
Use Rover for ci-groups

### PR DESCRIPTION
**Pull request by hongkailiu**
These groups have been created on rover and on clusters.
And this is a test run before merging https://github.com/openshift/release/pull/25401

/cc @openshift/test-platform

### CODE REVIEW

**Review by petr-muller — APPROVED** [review]
:heart:

**Comment by hongkailiu** [comment]
/hold

I will merge after some sanity checks.

**Pull request merged**
